cabal-install/Distribution/Client/ProjectPlanning/Types.hs

```diff
@@ -428,8 +427,10 @@ elabExeDependencyPaths elab =
 elabSetupDependencies :: ElaboratedConfiguredPackage -> [ConfiguredId]
 elabSetupDependencies elab =
     case elabPkgOrComp elab of
-        ElabPackage pkg    -> CD.setupDeps (pkgLibDependencies pkg)
-        ElabComponent comp -> compSetupDependencies comp
+        ElabPackage pkg -> CD.setupDeps (pkgLibDependencies pkg)
+        -- TODO: Custom setups not supported for components yet.  When
```

Comment:
Well I assume we (will eventually) want them to instead have a special exe dep on a setup component, which in turn track its deps normally? So this isn't commit back?